src/main/java/org/example/Variables.java

```diff
@@ -0,0 +1,5 @@
+package org.example;
+
+public class Variables {
```

Comment:
ALWAYS avoid using public static variables and setting/getting them from different classes, it created a massive wave of confusion. You're better off just passing the value down the chain of method calls from the top level down. Delete this class